Sources/CreateAPI/Generator/Declarations.swift

```diff
@@ -159,6 +158,8 @@ struct Property {
     var metadata: DeclarationMetadata?
     // A nested declaration required used as a property type
     var nested: Declaration?
+    // If the schema is inlined by `allOf`. This is currently used only for the generation of decoders.
+    var isInlined: Bool?
```

Comment:
Quick question, what is the difference in meaning of this property between when the value is `nil` and when it's `false`?

Reply:
I made this `isInlined` optional because this is only used for generating schemas but the struct `Property` is also used for the generation of Paths, therefore I thought setting `false` for this even though we don't know if it's actually inlined or not is kinda misleading. I defined `true` or `false` during the generation of the schemas because it needs to know if it is in `allOf` generation flow or not.

So the answer to the question would be `false` means it's not inlined and `nil` means we don't know if it's inlined or not and don't need to know either.